lib/valueflow.h

```diff
@@ -171,6 +172,8 @@ namespace ValueFlow {
         /** Is this value passed as default parameter to the function? */
         bool defaultArg;
 
+        long long indirect;
```

Comment:
heh.. a 64-bit value seems excessive.

I have thought about putting these type of data in a union so we can avoid some waste of space but that is not something you should do now in this PR.

Reply:
Using an union would be good. Although, `indirect` could apply to other value types as well.